utils/bazel/llvm-project-overlay/libc/BUILD.bazel

```diff
@@ -992,10 +999,15 @@ libc_support_library(
 libc_support_library(
     name = "__support_osutil_quick_exit",
     hdrs = ["src/__support/OSUtil/quick_exit.h"],
-    textual_hdrs = [
-        "src/__support/OSUtil/linux/quick_exit.h",
-        #TODO: add support for GPU quick_exit (isn't just in a header.)
-    ],
+    textual_hdrs = select({
+        "@platforms//os:macos": [
+            "src/__support/OSUtil/darwin/quick_exit.h",
+        ],
+        "//conditions:default": [
+            "src/__support/OSUtil/linux/quick_exit.h",
+            #TODO: add support for GPU quick_exit (isn't just in a header.)
```

Comment:
Note: I'm about to touch quick_exit in https://github.com/llvm/llvm-project/pull/85955. Not sure if that will cause a conflict here.

Reply:
i can wait for yours, there's no rush on this one